src/poprox_storage/repositories/accounts.py

```diff
@@ -55,6 +55,21 @@ def fetch_account_by_email(self, email: str) -> Account | None:
             return accounts[0]
         return None
 
+    def store_account(self, account: Account) -> UUID | None:
```

Comment:
Is it worth revising the `web` code that uses the following `store_new_account` to use this version instead?

Reply:
I'm not totally sure, but maybe? I left the other method there so this wouldn't break anything, but I do kinda like having methods that accept domain objects instead of accepting the relevant info and creating domain objects internally